api/orchestration.go

```diff
@@ -222,3 +260,53 @@ func (o *docDBOrchestrator) dbSubnetGroupCreate(ctx context.Context, name string
 
 	return nil
 }
+
+// startTask starts the flywheel task and receives messages on the channels.  in the future, this
+// functionality might be part of the flywheel library
```

Comment:
:copy: 🍝 😆

Reply:
it does feel like this should be part of flywheel though